core/include/keep.S

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2014, STMicroelectronics International N.V.
```

Comment:
Including `.h` files from assembly files are fine, in fact I prefer that over `.S` files. The only reason we're sometimes including `.S` files instead is when we define assembly macros, it seems it's impossible to define those in `.h` files.